rc/lsp.kak

```diff
@@ -162,6 +165,10 @@
     4: optional message
 } regex lsp_location_format ^\h*\K([^:\n]+):(\d+)\b(?::(\d+)\b)?(?::([^\n]+))
 
+declare-option -docstring "Number of indent levels to skip" int lsp_indent_guides_skip 0
+declare-option -docstring "Highlight current indent scope" bool lsp_indent_guides_highlight_focused true
```

Comment:
Is `lsp_indent_guides_highlight_focused` really necessary? Can't we control that already by setting IndentGuideFocused to the same as IndentGuid

Reply:
Yeah we can, and I like it - seems more elegant and simpler. Done!